### ISSUE TITLE
Specify maximum lock duration for raiden token payments

### ISSUE THREAD

**Issue by sangaman** [ExchangeUnion/xud#1173]
Once multihop payments with variable route choice (dependent on #1068) and lock durations are supported, we will need to be able to specify the maximum lock duration for the payment as we do currently with lnd using `cltvLimit`. This will likely require changes to both xud and raiden.

**Comment by kilrau** [comment]
This feat just got merged into Raiden https://github.com/raiden-network/raiden/pull/4697

Props @offerm
